[Bug] [Module Name] Bug title   mysql cdc to clickhouse   ILIKE '%allow_experimental_lightweight_delete%'. Expected PROFILES (version 20.8.3.18)

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

Caused by: org.apache.seatunnel.common.utils.SeaTunnelException: org.apache.seatunnel.connectors.seatunnel.clickhouse.exception.ClickhouseConnectorException: ErrorCode:[COMMON-08], ErrorDescription:[Sql operation failed, such as (execute,addBatch,close) etc...] - java.sql.SQLException: Code: 62, e.displayText() = DB::Exception: Syntax error: failed at position 15 ('ILIKE'): ILIKE '%allow_experimental_lightweight_delete%'. Expected PROFILES (version 20.8.3.18)

because my clickhouse  version 的太低not support allow_experimental_lightweight_delete？

### SeaTunnel Version

2.3.2

### SeaTunnel Config

```conf
env {
  execution.parallelism = 2
  job.mode = "STREAMING"
  checkpoint.interval = 2000
}

source {
  MySQL-CDC {
    result_table_name = "fake"
    parallelism = 1
    server-id = 5656
    username = "mysql"
    password = "jjY@xklby1025"
    table-names = ["txjm_ppf.t_bp_salepos_d"]
    base-url = "jdbc:mysql://192.168.208.100:3316/txjm_ppf"
  }
}



sink {
  Clickhouse {
    host = "192.168.12.34:8123"
    database = "mysql_JM"
    table = "t_bp_salepos_d"
    username = "default"
    password = "CK@123"
    # cdc options
    primary_key = "bill"
    support_upsert = true
    allow_experimental_lightweight_delete = true
  }
}
```


### Running Command

```shell
./bin/seatunnel.sh --config ./config/v2.streaming.mysql2ck -e local
```


### Error Exception

```log
Caused by: org.apache.seatunnel.common.utils.SeaTunnelException: org.apache.seatunnel.connectors.seatunnel.clickhouse.exception.ClickhouseConnectorException: ErrorCode:[COMMON-08], ErrorDescription:[Sql operation failed, such as (execute,addBatch,close) etc...] - java.sql.SQLException: Code: 62, e.displayText() = DB::Exception: Syntax error: failed at position 15 ('ILIKE'): ILIKE '%allow_experimental_lightweight_delete%'. Expected PROFILES (version 20.8.3.18)

because my clickhouse  version 的太低not support allow_experimental_lightweight_delete？
```


### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/release-note.md
+++ b/release-note.md
@@ -15,6 +15,7 @@
 ### Connectors
 - [Elasticsearch] Support https protocol & compatible with opensearch
 - [Hbase] Add hbase sink connector #4049
+- [Clickhouse] Fix clickhouse old version compatibility #5326
 ### Formats
 - [Canal]Support read canal format message #3950
 - [Debezium]Support debezium canal format message #3981

--- a/seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/client/ClickhouseSinkWriter.java
+++ b/seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/client/ClickhouseSinkWriter.java
@@ -208,7 +208,8 @@ public class ClickhouseSinkWriter
             }
             return false;
         } catch (SQLException e) {
-            throw new ClickhouseConnectorException(CommonErrorCode.SQL_OPERATION_FAILED, e);
+            log.warn("Failed to get clickhouse server config: {}", configKey, e);
+            return false;
         }
     }
 }

